kie-wb-parent/kie-wb-distribution-wars/kie-wb/pom.xml

```diff
@@ -22,6 +22,19 @@
       <type>war</type>
     </dependency>
 
+    <dependency>
+      <groupId>org.postgresql</groupId>
+      <artifactId>postgresql</artifactId>
+    </dependency>
```

Comment:
These dependencies are missing runtime scope. Is that intentional?

Reply:
Fixed